### ISSUE TITLE
GPS not detected

### ISSUE THREAD

**Issue by Anckero** [ImpulseRC/OSD#27]
Hi,

I'm having a hard time making my BN-180 module work with my Wolf PDB.
Here is my setup:
- KISS FC v1.03 FW1.3-RC36i (latest update available)
- Wolf PDB  Rev1 FW45.12 (latest update available)
- GPS BN-180

The GPS is wired correctly to the Wolf PDB, it flashes in blue and after a bit of time it flashes in red too (meaning it got a GPS fix).
Just in case, here is my wiring:
BN-180 red wire (3.0V-5.5V) → "5V" pad of the PDB (on top of the "GPS TX" pad)
BN-180 green wire (RX input) → "GPS TX" pad of the PDB
BN-180 white wire (TX output) → "GPS RX" pad of the PDB
BN-180 black wire (GND) → "Gnd" pad of the PDB (On top of the "Mag SCL" pad)

I turned on all the GPS items in the OSD but there is no data shown when I leave the OSD settings.

Could you tell me what I missed?
I noticed my FC is getting really hot when I plug the LiPo. The mAh count is going crazy fast!
I tried to unplug the GPS, same issue. Then I removed the 4 GPS wires from the PDB and no more overheating!
That's really strange because I checked and there is no shortcut between the GPS wire (without the GPS module connected to them).

A friend with a Kiss FC v2 and Wold PDB is facing the exact same issue with a BN-220 which is exactly the same hardware as my BN-180. No data and overheating.
I saw, here on GitHub, that you (the ImpulseRC devs) used a BN-180 in the development process so I really don't get why mine is not recognized.

Thanks in advance

**Comment by awolf78** [comment]
Can you post some pictures of your setup please? I also need to know what else is connected to the PDB and how.

**Comment by Anckero** [comment]
Hi awolf78,

I brought my setup to my work place and a colleague from the electronic team checked everything. We tried to wire up everything again and it worked.
The only difference was that I twisted the wires in the first place and my colleague told me that the power wires could create problem on the RX and TX wires. Now with strait cables the Wolf PDB and the GPS are talking to each other.
I will close this issue then, sorry for the inconvenience. :)
I didn't know that twisted wires could have this kind of effect.

Thanks for your support!
